spec/models/portfolio_spec.rb

```diff
@@ -0,0 +1,14 @@
+describe Portfolio do
+  let(:portfolio)       { FactoryBot.create(:portfolio) }
+  let(:portfolio_item)  { FactoryBot.create(:portfolio_item) }
+
+  context "#add_portfolio_item" do
```

Comment:
Prefer `describe` when describing the method under test.  `context` is more for when you have different setup conditions.